Sources/OpenTelemetryApi/Metrics/Stable/LongGaugeBuilder.swift

```diff
@@ -6,5 +6,7 @@
 import Foundation
 
 public protocol LongGaugeBuilder : AnyObject {
+    func setUnit(_ unit: String) -> LongGaugeBuilder
```

Comment:
setDescription & setUnit are accessible via the InstrumentBuilder protocol, remove these.